rm/rm-node/src/main/java/org/ow2/proactive/resourcemanager/utils/RMNodeStarter.java

```diff
@@ -681,6 +681,16 @@ public void run() {
         }));
     }
 
+    public static boolean simpleConfigureNodeForDataspace(Node node) throws ActiveObjectCreationException, NodeException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: org.objectweb.proactive.ActiveObjectCreationException, org.objectweb.proactive.core.node.NodeException [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1160)